### DISCUSSION TITLE
Blocked account

### DISCUSSION THREAD

**Discussion by jvvillanueva9104** [vercel/community#4396] [Account Issue]
How bad is your backlog? It's been over two weeks since I reached out to you guys about this problem and haven't heard back. 

I've worked as a Customer Service Manager before and unless you have 30,000 tickets in your backlog, I don't think that it should take this long to respond to a simple query. 

Either you're really behind or you're just disregarding "hobby" user queries 'cause they're just free which does not look good for your company.

**Comment by nuhptr** [comment]
I wait for 1 week, and whenever I want to deploy my personal website only pro and they don't care

**Comment by amyegan** [comment]
The support team is definitely not disregarding Hobby login issues. There is an unusually high number of tickets right now and they're being addressed as quickly as possible in alignment with our [support terms](https://vercel.com/legal/support-terms). 

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

**Reply by amyegan** [reply]
If you share the ticket number I can check the status for you

**Reply by nuhptr** [reply]
00164114 (1 week ago)

**Reply by Developsom** [reply]
@amyegan hey beautiful, i have the same problem for a while now
can you help me out?

**Reply by amyegan** [reply]
Thanks for sharing the case number @nuhptr. It looks like the support team solved the problem for you. Please let us know if you still have any trouble with your account.
